src/styles/styles.js

```diff
@@ -0,0 +1,67 @@
+import { css, Global, keyframes } from '@emotion/react'
```

Comment:
must be TS file, no JS.

Reply:
In future try better fileName. animation.ts

Reply:
updated filename